src/scripts/main.js

```diff
@@ -1,11 +1,21 @@
 'use strict';
 
 function waitFor(element, eventName) {
-  // write your code here
+  return new Promise((resolve, reject) => {
+    element.addEventListener(eventName, () => {
+      resolve(
+        `It was ${eventName} on the element: ${element.nodeName}, id: ${element.id}.`,
+      );
+    });
+  });
```

Comment:
The `waitFor` function correctly returns a Promise that resolves when the specified event occurs on the given element. This is a good use of Promises for handling asynchronous events.